### ISSUE TITLE
[2024] Super Consistency of Neural Network Landscapes and Learning Rate Transfer

### ISSUE THREAD

**Issue by gatheluck** [gatheluck/PaperReading#831]
## 基本情報
- 論文タイトル：Super Consistency of Neural Network Landscapes and Learning Rate Transfer
- 著者：Lorenzo Noci, Alexandru Meterez, Thomas Hofmann, Antonio Orvieto
- 公開日：2024/09/26
- リンク：
  - 論文 (OpenReview)：https://openreview.net/forum?id=rgwhJ7INtZ
  - コード (GitHub)：

### ベースとなった研究
- []()

### 影響を与えた最新の研究
- []()

## メモ
- []()
